docs/conf.py

```diff
@@ -250,6 +250,7 @@ def _get_rst_filepath_from_path(filepath: pathlib.Path):
         "baseoperator.py",
         "connection.py",
         "dag.py",
+        "dagrun.py"
```

Comment:
```suggestion
        "dagrun.py",
```

Reply:
ah! good catch! 😰 I did the change in the Github after finally figuring out why things weren't rendering.